### DISCUSSION TITLE
Night mode ?

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#2612] [Q&A]
hello! is there a night mode in SD Web Ui and if so how do I activate it?

**Comment by TheLastBen** [comment]
try adding `--theme dark`

**Reply by darth2602** [reply]
sorry, but, where ?

**Reply by darth2602** [reply]
where i must add  "--theme dark"  to have the dark mode ?  (i'm on google colab)

**Reply by darth2602** [reply]
je ne comprend pas ....a quel endroit je doit ajouter  "--theme dark" ???
